buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -7,4 +7,6 @@ object Versions {
     const val slf4j = "1.7.30"
     const val logback = "1.2.3"
     const val dockerJavaApi = "3.2.7"
+    const val h2 = "1.4.200"
+    const val kotlinReflect = "1.1.51"
```

Comment:
Doesn't reflect have the same version as language?

Reply:
Fixed